### ISSUE TITLE
Player names

### ISSUE THREAD

**Issue by Nocte-** [Nocte-/hexahedra#14]
Allow players to enter their name, and display this name over their heads.

We can do this in a hacky way until we have a proper login system, but the name tag should be a component in the entity system so we can reuse it for other purposes as well.

**Comment by ghost** [comment]
I see NPC's on the horizon.

**Comment by Nocte-** [comment]
Yeah I know, I still need to hook up the occlusion queries for the name tags. ;)

Player models and physics already work as ES components, and can be manipulated in Lua (jumping is implemented this way). NPCs definitely aren't far off.
